lib/matplotlib/units.py

```diff
@@ -49,6 +49,10 @@ def default_units(x, axis):
 from matplotlib import cbook
 
 
+class ConversionError(ValueError):
```

Comment:
this is a bit bikesheddy but it looks more like a TypeError than a ValueError to me.